rpm/qm.spec

```diff
@@ -59,6 +59,11 @@ BuildArch: noarch
 BuildRequires: golang-github-cpuguy83-md2man
 BuildRequires: container-selinux
 BuildRequires: make
+%if 0%{?centos} || 0%{?rhel}
+# Add EPEL repository for RHEL 9 CentOSStream
+BuildRequires: epel-release
```

Comment:
Probably won't help. Would be much easier to use `epel-` targets for copr as those can be used by TMT for rhel and centos tests. It's currently being changed in https://github.com/containers/qm/pull/298/files#diff-2b4300f6d69654fd71c0cdd329b5a5f13fa8fe7eb709c76276bdfd3ea1dde578R23 .

Reply:
@dougsland please refer this #298 it is playground, at least to build the rpm